app/views/build_mailer/build_break_email.html.haml

```diff
@@ -22,6 +22,7 @@
         %li
           = link_to("Part number #{failed_build_part.id}", repository_build_part_url(@build.repository, @build, failed_build_part))
           = failed_build_part_sentence(failed_build_part)
+        = "- #{failed_build_paths(failed_build_part)}"
```

Comment:
a bit of a nit but would you mind making this text smaller and maybe a shade of grey?